invoiceregistry.go

```diff
@@ -409,6 +411,16 @@ func (i *InvoiceRegistry) invoiceEventLoop(ctx context.Context) error {
 				break
 			}
 
+			if state.autoSettle {
+				select {
```

Comment:
this select construct might be extracted now

Reply:
done